test/classBasedPlugins/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "target": "es6",
+    "module": "commonjs",
+    "strict": false,
+    "experimentalDecorators": true
```

Comment:
I _think_ I've heard they are going to change this in the next version of TS. When will decorators be out of experimental? Have you got any info?

Reply:
No, I got no information on this.
However I think we will know in two days: https://github.com/microsoft/TypeScript/issues/30555

Reply:
Decorators are still experimental in typescript 3.5.x.